lib/cadet_web/controllers/auth_controller.ex

```diff
@@ -0,0 +1,171 @@
+defmodule CadetWeb.AuthController do
+  use CadetWeb, :controller
+
+  import Ecto.Changeset
+
+  use PhoenixSwagger
+
+  alias Cadet.Accounts
+  alias Cadet.Auth.Guardian
+  alias Cadet.Accounts.Form
+
+  def create(conn, %{"login" => attrs}) do
+    changeset = Form.Login.changeset(%Form.Login{}, attrs)
+
+    if changeset.valid? do
+      login = apply_changes(changeset)
+
+      case Accounts.sign_in(login.email, login.password) do
+        {:ok, user} ->
+          {:ok, access_token, _} =
+            Guardian.encode_and_sign(user, %{}, token_type: "access", ttl: {1, :hour})
+
+          {:ok, refresh_token, _} =
+            Guardian.encode_and_sign(user, %{}, token_type: "refresh", ttl: {52, :weeks})
+
+          conn
+          |> render("token.json", access_token: access_token, refresh_token: refresh_token)
+
+        {:error, _reason} ->
+          conn
+          |> send_resp(403, "Wrong email and/or password")
+      end
+    else
+      conn
+      |> send_resp(404, "Missing parameters")
+    end
+  end
+
+  def create(conn, _params) do
+    conn
+    |> send_resp(404, "Missing parameters")
```

Comment:
I think for one function call we shouldn't use pipe syntax, what do you think?

Reply:
Sure. Maybe we can base our code styling on this guideline: https://github.com/christopheradams/elixir_style_guide ?